splunk/common-files/createdefaults.py

```diff
@@ -25,7 +25,9 @@
 splunk_hec_token = os.environ.get("SPLUNK_HEC_TOKEN", None)
 splunk_password = os.environ.get("SPLUNK_PASSWORD", None)
 splunk_idxc_secret = os.environ.get("SPLUNK_IDXC_SECRET", None)
+splunk_idxc_pass4SymmKey = os.environ.get("SPLUNK_SHC_PASS4SYMMKEY", None)
```

Comment:
Should be `SPLUNK_IDXC_PASS4SYMMKEY`

Reply:
https://www.youtube.com/watch?v=atGjfmqb_TE